definitions/contested/json/CaseEvent/CaseEvent.json

```diff
@@ -442,7 +443,7 @@
     "DisplayOrder": 37,
     "PreConditionState(s)": "judgeDraftOrder",
     "PostConditionState": "scheduleRaiseDirectionsOrder",
-    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/case-orchestration/move-collection/draftDirectionDetailsCollection/to/draftDirectionDetailsCollectionRO",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/case-orchestration/draft-hearing-order/judge",
```

Comment:
Sorry Jakub, this endpoint was changed in FR-1855 as well so looks like we'll have a conflict and i assume in COS we'll have done similar stuff moving the move collection logic out of the controller it was in.
FR-1855 is in sign off and not merged yet. If things in it need changed to better work with this i'd be happy to do so and move it back to testing.
Sorry i didn't realise these were touching the same area earlier

Reply:
no probs, thanks for the heads up